release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -156,6 +156,10 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-9-olm"]
 === Operator lifecycle
 
+[id="ocp-4-9-olm-k8s-1.22"]
+==== Operator Lifecycle Manager (OLM) upgraded to Kubernetes 1.22
+
+Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM) supports Kubernetes 1.22. As a result, xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[a significant number of `v1beta1` APIs have been removed and updated to `v1`]. Before upgrading a cluster to {product-title} {product-version}, cluster administrators should xref:../operators/admin/olm-upgrading-operators.adoc[update their installed Operators to the latest channel].
```

Comment:
Should we include mention that if any operators that still depend on those removed apis (particularly the v1beta1 version of the CustomResourceDefinition API) are running on your cluster, they will be unable to run on 4.9?

Reply:
@kevinrizza PTAL and let me know if these changes work for you.